src/components/CardDiversity/cardDiversity.js

```diff
@@ -0,0 +1,27 @@
+import React, { memo } from 'react';
```

Comment:
Als ik kijk naar de rest van de codebase dan is een component altijd PascalCase. 

Let op: Het kan zijn wanneer je de filename aanpast lokaal dat die niet wordt opgepikt door git.

Op te lossen door het volgende te gebruiken:

git mv cardDiversity.js tmp.js
git mv tmp.js CardDiversity.js